homeassistant/components/axis/device.py

```diff
@@ -77,12 +76,6 @@ def serial(self):
         """Return the serial number of this device."""
         return self.config_entry.unique_id
 
-    @property
-    def option_camera(self):
-        """Config entry option defining if camera should be used."""
-        supported_formats = self.api.vapix.params.image_format
-        return self.config_entry.options.get(CONF_CAMERA, bool(supported_formats))
```

Comment:
Was `CONF_CAMERA` option removed earlier from the options flow?

Reply:
No it was never a part of the options flow. Its a remnant from before options flow was created, I had a plan for how to configure which is not aligned with how things are supposed to be done today. The way to do it now should be to use disable the entity instead

Reply:
Ok, thanks!